pkg/asset/installconfig/azure/azure.go

```diff
@@ -17,9 +17,6 @@ import (
 	azsub "github.com/Azure/azure-sdk-for-go/profiles/latest/resources/mgmt/subscriptions"
 )
 
-const (
```

Comment:
This should probably stay.

Reply:
I am removing the default region because it would have to be maintained for every new environment.